hbase-client/src/main/java/org/apache/hadoop/hbase/UnknownRegionException.java

```diff
@@ -26,12 +26,25 @@
  */
 @InterfaceAudience.Public
 public class UnknownRegionException extends DoNotRetryRegionException {
-  private static final long serialVersionUID = 1968858760475205392L;
+
+  private static final long serialVersionUID = 8052818407096543141L;
```

Comment:
the UID changed here because the serialized `cause` member will be something other than `null` now?

Reply:
Yes, although truthfully I didn't think too much about it, just used the tool to regen it.

Reply:
Do we really need to change it?

Reply:
Nope. I reverted it back.